zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -36,6 +39,9 @@ pub enum WalletMigrationError {
 
     /// Wrapper for rusqlite errors.
     DbError(rusqlite::Error),
+
+    /// Wrapper for amount balance violations
+    BalanceError(BalanceError),
```

Comment:
When was `WalletMigrationError` added? Should this extra case be added to the changelog?

Reply:
Added.